framework/math/ParallelVector/ParallelSTLVector.h

```diff
@@ -91,6 +96,10 @@ class ParallelSTLVector : public ParallelVector
    * compatible.*/
   void CopyLocalValues(const ParallelVector& y) override;
 
+  /**Sets the local values of the vector equal to that of the PETSc vector.
+  * The sizes must be compatible.*/
+  void CopyLocalValues(Vec y) override;
```

Comment:
Wouldn't be a terrible idea to go ahead and implement a PETSc version of BlockCopyLocalValues if it isn't too difficult.

Reply:
Done